docs/guides/main.rst

```diff
@@ -9,10 +9,9 @@ collection better.
 Installing
 ----------
 
-You will need Python.
-Beets works on `Python 2.7`_ and Python 3.4 or later.
+You will need `Python 3.6`_. But Beet also works with Python 2.7 and later.
```

Comment:
Python 3.4 actually works fine. 3.6 is just what we recommend, especially for Windows.